sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -55,8 +55,36 @@ typedef struct
    * The model ID used to identify the capabilities of a device based on the Digital Twin document.
    */
   az_span model_id;
+
+  /**
+   * The data_format is sent in the username to inform IoT Hub what format the device expects Twin
```

Comment:
What is this `data_format` referring to? Is this some object we define? I am asking because I am trying to understand why we have the underscore in it?

Reply:
Left over from an earlier version! It should be method_twin_content_type.  Updating.